### PR TITLE
Move qpid scripts ({gutterball,server}/bin/qpid)

### PR DESCRIPTION
**Pull request by kahowell**
I also removed gutterball references from the script.

Also, I found that `-q` to `server/bin/deploy` is sufficient to enable
qpid after running `server/bin/qpid/configure-qpid.sh`.

This should be followed up by an update to candlepinproject.org docs.

### CODE REVIEW

File: server/spec/candlepin_scenarios.rb [hunk 1]
```diff
@@ -655,8 +655,8 @@ def do_export(client, dest_dir, opts={}, uuid=nil)
 class CandlepinQpid
   def initialize() 
     @address='amqps://localhost:5671/allmsg'
-    @gbcrt = "gutterball/bin/qpid/keys/qpid_ca.crt"
-    @gbkey = "gutterball/bin/qpid/keys/qpid_ca.key"
+    @gbcrt = "server/bin/qpid/keys/qpid_ca.crt"
```
Comment (barnabycourt) [inline server/spec/candlepin_scenarios.rb]: This should probably be called something other than "gbcrt"

**Comment by barnabycourt** [comment]
@kahowell please update the deploy script to automatically generate the certs & configure everything if the -q (qpid) configuration is used.

**Comment by kahowell** [comment]
298434e..54035c8:
 - `deploy -q` invokes qpid configure script
 - `deploy -q -f` cleans qpid certs and then regenerates
 - `{gb -> qpid_}{crt,key}`

**Comment by barnabycourt** [comment]
@kahowell running in a vagrant box. server/bin/deploy -q -a -f. 

I get errors running the rspec:qpid_spec

 1) Qpid Broker flow stop is detected
     Failure/Error: skip("Qpid keys not found, skipping Qpid spec tests") if @cq.no_keys
     
     TypeError:
       no implicit conversion of nil into String
     # ./server/spec/candlepin_scenarios.rb:663:in `file?'
     # ./server/spec/candlepin_scenarios.rb:663:in `no_keys'
     # ./server/spec/qpid_spec.rb:21:in `block (2 levels) in <top (required)>'

  2) Qpid Broker transition to SUSPEND mode
     Failure/Error: skip("Qpid keys not found, skipping Qpid spec tests") if @cq.no_keys
     
     TypeError:
       no implicit conversion of nil into String
     # ./server/spec/candlepin_scenarios.rb:663:in `file?'
     # ./server/spec/candlepin_scenarios.rb:663:in `no_keys'
     # ./server/spec/qpid_spec.rb:21:in `block (2 levels) in <top (required)>'

  3) Qpid Broker reconnect should work
     Failure/Error: skip("Qpid keys not found, skipping Qpid spec tests") if @cq.no_keys
     
     TypeError:
       no implicit conversion of nil into String
     # ./server/spec/candlepin_scenarios.rb:663:in `file?'
     # ./server/spec/candlepin_scenarios.rb:663:in `no_keys'
     # ./server/spec/qpid_spec.rb:21:in `block (2 levels) in <top (required)>'

**Comment by barnabycourt** [comment]
@kahowell One other comment. Could the qpid config script check to see if qpid is already installed and not re-run the yum steps?

**Comment by kahowell** [comment]
54035c8..168fcfc:
 - s/qpid_cert/qpid_crt - doh!
 - fix logic error in configure script so we don't get yum install every time

@barnabycourt that should address all of latest round of feedback.

**Comment by barnabycourt** [comment]
looks good.

**Pull request merged**